dumpling/tests/naughty_strings/naughty_strings.escape-schema.sql

```diff
@@ -1,4 +0,0 @@
-CREATE TABLE `escape` (
-  `a` text CHARACTER SET utf8mb4 COLLATE utf8mb4_bin
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
```

Comment:
why delete this file?

Reply:
Because it's never used.

Reply:
seems it still used in test https://github.com/pingcap/tidb/blob/321d307f3b23877fb2be0e7534a9ecd6a655be05/dumpling/tests/naughty_strings/run.sh#L23

Reply:
This file's path is `dumpling/tests/naughty_strings/naughty_strings.escape-schema.sql` 🤔

Reply:
ohohoh my bad... two file in different with same name